doppelganger/design.md

```diff
@@ -0,0 +1,15 @@
+# Doppelganger Protection - Design Overview
+
+This document provides an overview of the Doppelganger Protection mechanism in `ssv.network`.
+
+🔗 **Full Documentation**: [Doppelganger Protection Design on HackMD](https://hackmd.io/@TjjCyxe9Qxm0hOn3HZ_YIA/HygAf4M5yg)
```

Comment:
lets include the full doc here. no need to link to outside of the repo where there's no version tracking